engine-dmn/engine/src/main/java/org/camunda/bpm/dmn/engine/impl/DefaultDmnEngine.java

```diff
@@ -112,7 +112,7 @@ public DmnDecisionTableResult evaluateDecisionTable(DmnDecision decision, Variab
     if (decision instanceof DmnDecisionImpl && decision.isDecisionTable()) {
       DefaultDmnDecisionContext decisionContext = new DefaultDmnDecisionContext(dmnEngineConfiguration);
 
-      DmnDecisionResult decisionResult = decisionContext.evaluateDecision(decision, variableContext);
+      DmnDecisionResult decisionResult = decisionContext.evaluateDecision(decision, variableContext, null);
```

Comment:
@yanavasileva This will not change because we need to pass the decisionInsanceId to `decisionContext.evaluateDecision(decision, variableContext, null);`

Reply:
When you are ready, please submit your changes, I will have a look then.